docker.local/build.unit_test/Dockerfile

```diff
@@ -4,14 +4,17 @@ ENV GO111MODULE=on
 
 # Download the dependencies:
 # Will be cached if we don't change mod/sum files
+COPY ./code/go/0chain.net/go.mod $SRC_DIR/go/0chain.net/
+COPY ./code/go/0chain.net/go.sum $SRC_DIR/go/0chain.net/
+
+RUN cd $SRC_DIR/go/0chain.net && go mod download -x
+RUN go get github.com/vektra/mockery/v2/.../
+
 COPY ./code/go/0chain.net $SRC_DIR/code/go/0chain.net
 COPY ./generate_mocks.sh $SRC_DIR/generate_mocks.sh
 
-RUN go get github.com/vektra/mockery/v2/.../
 RUN cd $SRC_DIR && ls && ./generate_mocks.sh
 
-RUN cd $SRC_DIR/code/go/0chain.net && \
-    go mod download
 
```

Comment:
copy gozstd from zbuild_base, it will save some time to build it again. https://github.com/0chain/0chain/blob/556fedc2aad08b9efeb0b4901ad377d2928cd611/docker.local/build.sharder/Dockerfile#L19

Reply:
will it be faster if we move existing code upper, so we could cache it?

Reply:
yes. `move upper` should be a solution too.